PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -111,7 +111,9 @@ public class PanModalPresentationController: UIPresentationController {
             view = DimmedView()
         }
         view.didTap = { [weak self] _ in
-            self?.dismissPresentedViewController()
+            if self?.presentable?.allowsTapToDismiss ?? true {
```

Comment:
could we change this to: `self?.presentable?.allowsTapToDismiss == true` (makes the logic easier to read)